apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/GroupInfoModal.tsx

```diff
@@ -0,0 +1,143 @@
+import { useCallback, useState } from 'preact/compat';
+import { Alert, Button, Modal } from 'react-bootstrap';
+
+const defaultClosedOnly = true;
+
+export function GroupInfoModal({
+  modalFor,
+  numOpenInstances,
+  show,
+  onHide,
+  onSubmit,
+}: {
+  modalFor: 'all' | 'selected' | 'ungrouped';
+  numOpenInstances: number;
+  show: boolean;
+  onHide: () => void;
+  onSubmit: (closedOnly: boolean) => void;
+}) {
+  const [closedOnly, setClosedOnly] = useState(defaultClosedOnly);
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const handleClose = useCallback(() => {
+    onHide();
+    setClosedOnly(defaultClosedOnly);
+    setIsSubmitting(false);
```

Comment:
Nit, `Modal` offers an `onExited` callback that can be useful for cleanup like this. That has the (minor) benefit of ignoring a flash of default content as the modal is animating out.